azure-pipelines.yml

```diff
@@ -19,7 +19,7 @@ stages:
 
   - job: security_analysis
     displayName: Security Analysis
-
+    condition: not(variables['SKIP_MSAZURE'])
```

Comment:
why not skip_terraform?

Reply:
there're 2 think to skip - terraform and credscan. I can separate to 2 variables but I thought that it would be simpler this way.

Reply:
I agree a single variable is better, also not sure about the name.
Maybe: `SKIP_UNSUPPORTED_PIPELINE_COMPONENTS`?